person.rb

```diff
@@ -18,4 +18,15 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
+
+  def correct_name
+    @name.capitalize!
+  end
```

Comment:
- [ ] kindly make sure to make `Person` class inherits from `Nameable` class. It also has a method `correct_name`Inheritance allows you   to inherit the characteristics of one class into another class.
please fix this  example below
``class Person < Nameable``